Resources/public/js/views/ez-testoneintroview.js

```diff
@@ -0,0 +1,93 @@
+YUI().add('ez-testoneintroview', function(Y) {
+    'use strict';
+    /**
+     * Provides the intro subview of test one
+     *
+     * @module ez-testoneintroview
+     */
+    Y.namespace('eZ');
+
+    var INTRO_CLASS_PREFIX = 'ez-view-testoneintroview',
+        LOADER = Y.Node.create('<span class="ez-inline-loader"></span>');
+
+    /**
+     * The intro view
+     *
+     * @namespace eZ
+     * @class TestOneIntroView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.TestOneIntroView = Y.Base.create('testOneIntroView', Y.eZ.TemplateBasedView, [], {
+        initializer: function () {
+            this.on('introLoaded', this._introLoaded);
+            this.on('introLoading', this._introLoading);
+            this.on('introLoadFailed', this._introLoadFailed);
+        },
+
+        /**
+         * Event handler for succesful intro loaded
+         *
+         * @method _introLoaded
+         * @param {Object} e event facade
+         * @protected
+         */
+        _introLoaded: function (e) {
+            var container = this.get('container');
+            LOADER.detach();
+            container.removeClass(INTRO_CLASS_PREFIX + '--loading');
+            this.set('intro', e.intro);
+            this.render();
+        },
+
+        /**
+         * Event handler for starting intro load
+         *
+         * @method _introLoading
+         * @param {Object} e event facade
+         * @protected
+         */
+        _introLoading: function (e) {
+            var container = this.get('container');
+            container.empty();
+            container.prepend(LOADER);
+            container.addClass(INTRO_CLASS_PREFIX + '--loading');
+        },
+
+        /**
+         * Event handler for intro load failure
+         *
+         * @method _introLoadFailed
+         * @param {Object} e event facade
+         * @protected
+         */
+        _introLoadFailed: function (e) {
+            var container = this.get('container');
+            LOADER.detach();
+            container.removeClass(INTRO_CLASS_PREFIX + '--loading');
+        },
+
+        render: function () {
+            var intro = this.get('intro');
+
+            this.get('container').setHTML(this.template({
+                intro: (intro ? intro.render().get('container').getHTML() : null)
```

Comment:
Przy okazji, to wydaje się trochę dziwne. Widok nazywa się `testOneIntroView` a atrybut `intro` jest instancją widoku. To wygląda jakby widok trzymał samego siebie w swoim atrybucie.